tobago-theme/tobago-theme-standard/src/main/npm/ts/tobago-split-layout.ts

```diff
@@ -81,11 +81,13 @@ class SplitLayout extends HTMLElement {
   move(event: MouseEvent): void {
     event.preventDefault();
     const data = SplitLayoutMousedown.load();
-    const previousArea = data.previous;
-    if (this.orientation === "horizontal") {
-      previousArea.style.width = String(event.pageX - this.offset) + "px";
-    } else {
-      previousArea.style.height = String(event.pageY - this.offset) + "px";
+    if(data.previous !== null) {
```

Comment:
change this 2 lines: first put into variable and than check, if null (might be faster)